packages/fetchai/skills/echo/handlers.py

```diff
@@ -30,11 +30,6 @@ class EchoHandler(Handler):
 
     SUPPORTED_PROTOCOL = DefaultMessage.protocol_id
 
-    def __init__(self, **kwargs):
```

Comment:
Why has this been removed? Then we need to update the docs as well

Reply:
It was a mistake, I'll reintroduce them